types/font-carrier/tslint.json

```diff
@@ -0,0 +1,13 @@
+{
+    "extends": "@definitelytyped/dtslint/dt.json",
+    "rules": {
+        "array-type": false,
```

Comment:
Please don't disable this or any other lint rule for new definitions, thanks~

Reply:
hello， I have dealt with the problem.